### DISCUSSION TITLE
Need for a readme.md file and syntax example on the main git site

### DISCUSSION THREAD

**Discussion by Kashina69** [TEPL-Team/TEPL#14] [Ideas]

**Comment by Xtqm** [comment]
Adding a readme.md file would be useful. I will attend to it as soon as I can! 
What do you mean by the main git site?

**Comment by Kashina69** [comment]
Braa my english bad I mean the main page of your project where readme.md
file output automatically is shown

**Comment by Xtqm** [comment]
Oh ok lol. Thanks for the suggestion! I'll get to work soon.

**Comment by Kashina69** [comment]
Bro i have started applying and made this resume changing my portfolio
website too but ya it will take some time but can you review my resume and
give feedback? plzz

**Comment by Xtqm** [comment]
Could you please tell me what the repo is called or just give me the link please? Otherwise, I can't help.

**Comment by Kashina69** [comment]
https://docs.google.com/document/d/1Mj5LT9suwxHar8JxwNnjMrn147Z4LhuxfuqZuJiWMAE/edit

this is the good drive (docs ) link of my latest resume 

and can you also check my portfolio plzzz
it's
https://princerawat-webdev-portfolio.web.app/

**Comment by Kashina69** [comment]
plzz tell sir

**Comment by Xtqm** [comment]
I do not feel entirely secure opening a google doc, so I am sorry, but I cannot open it. As for you profile, it is good, but has a few things you should fix. Your *Design Only* page is somewhat messy and doesn't have very good readability, I'd fix that. Links to things like your weather app inside your *side projects* website, don't take me to that project. They leave me at the top. Make sure your ID's and other parts of your code are working properly and are exact. 

Like always, Happy Coding.

**Comment by Kashina69** [comment]
Thxx for the review under your security concern
